src/DateTimePicker.jsx

```diff
@@ -250,7 +251,8 @@ DateTimePicker.propTypes = {
 		React.PropTypes.string
 	]),
 	dateOnly: React.PropTypes.bool,
-	forceCalendar: React.PropTypes.bool
+	forceCalendar: React.PropTypes.bool,
+	onChangeCallback: React.PropTypes.func
```

Comment:
Yeah, that's the only thing about this. This element is relying on a state change being relayed on change rather than an event happening.